zerver/openapi/python_examples.py

```diff
@@ -344,6 +344,25 @@ def create_reusable_invitation_link(client: Client) -> None:
     validate_against_openapi_schema(result, "/invites/multiuse", "post", "200")
 
 
+@openapi_test_function("/invites/{prereg_id}:delete")
+def revoke_email_invitation(client: Client) -> None:
+    request = {
+        "invitee_emails": "harry@zulip.com",
+        "invite_expires_in_minutes": 60 * 24 * 10,  # 10 days
```

Comment:
Maybe make the email more specific `delete-invite@zulip.com` and you could just use the default 1440 minutes here, since it's not showing up in the rendered documentation.